common/djangoapps/student/handlers.py

```diff
@@ -1,34 +0,0 @@
-"""
-Handlers for student
-"""
```

Comment:
Everything looks good except I'm confused about where this module was ever getting loaded from!

Reply:
I actually could never get this code to work which may be because it wasn't hooked up to anything ><

Reply:
https://github.com/search?q=repo%3Aopenedx%2Fedx-platform%20%22common.djangoapps.student.handlers%22&type=code It doesn't seem to have been called anywhere

Reply:
Haha, I was wondering if this was the same event that was giving you trouble! There's no code reference to "handlers" anywhere in `common/djangoapps/student/` and it looks like all the other apps load their signal receivers from their ready method (rather than from some hypothetical non-local mechanism).

@kiram15 It looks like this code was never hooked up to send to the event bus. I don't see any log entries for `"Producing unenrollment-event event via"` either, which is pretty good confirmation.

However...I _do_ see log entries for `"Responses of the Open edX Event <org.openedx.learning.course.unenrollment.completed.v1>" source="/edx/var/log/lms/edx.log"` pretty steadily over the past month. So... _something_ was sending it to the event bus!

Reply:
Not necessarily. Those are the logs we get for local signal sends

Reply:
Ah right, thanks! Searching for the topic is more appropriate... and I only find the stage-prefixed topic, not the production one. So that checks out.